src/worldstate/include/routines/BottomLine/FindLine.h

```diff
@@ -0,0 +1,34 @@
+//
+// Created by drdc-s2632 on 17/07/18.
+//
+
+#ifndef PROJECT_FINDLINE_H
+#define PROJECT_FINDLINE_H
+
+#include "State.h"
+#include <constants.h>
+#include <line_detect/LineDetectMsg.h>
+
+/*** Communicating Class {findLine, adjustToLine, followLine} ***/
+class FindLine : public State {
+  public:
+    FindLine(std::unordered_map<std::string, double>* constants)
+      : State(constants) {}
+    std::vector<ros::Subscriber>
+    getNodeSubscriptions(ros::NodeHandle nh) override;
+
+  private:
+    ros::Timer timer_;
+
+    /**
+     * Decides based on image data whether the robot still needs to
```

Comment:
copy-paste leftovers

Reply:
/**
     * Decides based on image data whether the robot still needs to
     * align with the line.
     *
fixed